pkg/adapter/containers.go

```diff
@@ -257,6 +257,9 @@ func (r *LocalRuntime) Log(c *cliconfig.LogsValues, options *libpod.LogOptions)
 func (r *LocalRuntime) CreateContainer(ctx context.Context, c *cliconfig.CreateValues) (string, error) {
 	results := shared.NewIntermediateLayer(&c.PodmanCommand, false)
 	ctr, _, err := shared.CreateContainer(ctx, &results, r.Runtime)
+	if err != nil {
+		return "", err
+	}
 	return ctr.ID(), err
```

Comment:
For clarity this should be
return ctr.ID(), nil

Reply:
fixed